src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="product-card">
+      <img
+        src="styles/images/imac.jpeg"
+        alt="imac"
+        class="product-card__picture-imac"
+      >
+      <p class="product-card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+      <div class="stars" data-qa="card">
+        <div class="stars__content">
+          <div class="stars__stars-4">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-inactive"></div>
+          </div>
+          <div class="stars__text">
+            <p class="stars__text-reviews">Reviews: 5</p>
+          </div>
+        </div>
+      </div>
+      <div class="price">
+        <div class="price__text">
+          <p>Price</p>
+        </div>
```

Comment:
you could have avoided doing so, just this way 
```
<p class="price__text">
  Price
</p>
```
you don't use it – thus, you don't need it